tomcat/src/main/java/org/apache/coyote/controller/LoginController.java

```diff
@@ -0,0 +1,63 @@
+package org.apache.coyote.controller;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import nextstep.jwp.db.InMemoryUserRepository;
+import nextstep.jwp.model.User;
+import org.apache.catalina.Session;
+import org.apache.coyote.controller.utils.PathFinder;
+import org.apache.coyote.http11.httpmessage.common.ContentType;
+import org.apache.coyote.http11.httpmessage.request.Request;
+import org.apache.coyote.http11.httpmessage.request.requestbody.RequestBodyContent;
+import org.apache.coyote.http11.httpmessage.response.Response;
+
+public class LoginController extends AbstractController {
+
+    @Override
+    public void service(Request request, Response response) throws Exception {
+        if (request.isPostMethod()) {
+            doPost(request, response);
+            return;
+        }
+        if (request.isGetMethod()) {
+            doGet(request, response);
+        }
```

Comment:
ㅎㅎ 이렇게는 어떤가요?? 

(딱히 읽어가는데 있어서 어렵거나 큰 부분은 아니에요!)

```suggestion
        if (request.isPostMethod()) {
            doPost(request, response);
            return;
        }
        doGet(request, response);
```

Reply:
감사해요 필즈~. 해당 부분 if문 밖에서 doGet하는걸로 고쳤습니다. 😄